src/modules/network/components/DialogAddNetwork/DialogAddNetwork.js

```diff
@@ -127,9 +127,7 @@ const DialogAddNetwork = () => {
             <Input
               size="l"
               label="Service URL"
-              placeholder={t(
-                'Enter service URL, e.g. https://service.lisk.com or http://localhost:9901'
-              )}
+              placeholder={t('Enter service URL, e.g. https://testnet-service.lisk.com')}
```

Comment:
Since testnet has been added back to the list ot networks, wouldn't that imply that the placeholders should have custom node values rather than testnet values?

Reply:
Since Its just and example(because of `e.g`) @ikem-legend I don't think its a problem